configs/raspberrypi_config

```diff
@@ -91,6 +91,12 @@ function platform_modify() {
     run_and_log_cmd "sudo ln -fs /uboot/overlays work/rootfs/boot/overlays"
     run_and_log_cmd "sudo ln -fs /uboot/$CMDLINE work/rootfs/boot/$CMDLINE"
 
+    # Raspberry Pi headless configuration expects boot partition to be mounted
+    # at /boot, so replace these services with /uboot
+    run_and_log_cmd "find work/rootfs/lib/ -type f -name *.service | xargs sed -i.bak 's|/boot/|/uboot/|g'"
+    log_info "Diff from modified service files:\n$(find work/rootfs/lib/ -type f -name *.service | xargs -I@ bash -c 'diff @.bak @')"
```

Comment:
`diff -u` maybe?

Also, is there a `log_debug`?

Reply:
`diff -u` was even more verbose, so I went for the simple one. But as I move the output to the log file I can restore `-u` here.